### PR TITLE
initial commit for week 2 homework

### PR DESCRIPTION
**Pull request by MirSzabo**

### CODE REVIEW

File: databases/week2/week2exercise1.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+user_id
+) VALUES (
+"Learn about MySQL",
+"interesting",
+NOW(),
+NOW(),
+"2019-12-23 10:13:42",
+1,
+7
+);
```
Comment (flemeur) [inline databases/week2/week2exercise1.sql]: I would recommend that you indent the column names and the values just like you would inside a javascript function. 

Also while double quotes work for strings in MySQL they wont work in other databases. The standard way of specifying strings is using single quotes.

File: databases/week2/week2exercise2.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
+-- Create a new database containing the following tables:
+-- Class: with the columns: id, name, begins (date), ends (date)
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+CREATE DATABASE school;
+USE school;
+CREATE TABLE `class` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+`name` varchar(255) NOT NULL,
+`begins (date)` DATETIME NOT NULL DEFAULT NOW(),
+`ends (date)` DATETIME NOT NULL DEFAULT NOW(),
```
Comment (flemeur) [inline databases/week2/week2exercise2.sql]: I would not recommend putting spaces and parenthesis in column names. I would just call the two DATETIME columns `begins` and `ends`

Comment (MirSzabo) [reply databases/week2/week2exercise2.sql]: I updated the file with columns begins and ends

File: databases/week2/week2exercise1.sql (hunk 1 above)
Comment (MirSzabo) [reply databases/week2/week2exercise1.sql]: I updated the file with single quotes.

**Pull request merged**